pkg/authz/object/service.go

```diff
@@ -45,26 +44,12 @@ func (svc ObjectService) Create(ctx context.Context, objectSpec ObjectSpec) (*Ob
 	return newObject.ToObjectSpec(), nil
 }
 
-func (svc ObjectService) DeleteByObjectTypeAndId(ctx context.Context, objectType string, objectId string) (*wookie.Token, error) {
-	var newWookie *wookie.Token
-	err := svc.Env().DB().WithinTransaction(ctx, func(txCtx context.Context) error {
-		err := svc.Repository.DeleteByObjectTypeAndId(txCtx, objectType, objectId)
-		if err != nil {
-			return err
-		}
-
-		newWookie, err = svc.WarrantSvc.DeleteRelatedWarrants(txCtx, objectType, objectId)
-		if err != nil {
-			return err
-		}
-
-		return nil
-	})
-
+func (svc ObjectService) DeleteByObjectTypeAndId(ctx context.Context, objectType string, objectId string) error {
+	err := svc.Repository.DeleteByObjectTypeAndId(ctx, objectType, objectId)
```

Comment:
Shouldn't this happen within a transaction?

Reply:
No, since it's a single operation (which should be atomic)